src/memex/parse_document_claims.py

```diff
@@ -101,7 +101,10 @@ def transform_meta_(document_meta_dicts, items, path_prefix=None):
                 if not isinstance(value, list):
                     value = [value]
 
-                value = [v.strip() for v in value]
+                value = [v.strip() for v in value if v and v.strip()]
+
+                if not value:
+                    continue
```

Comment:
Isn't this now being applied to all meta objects, and not just the ones of type `title`?